src/stream.iced

```diff
@@ -0,0 +1,56 @@
+stream = require('stream')
+enc = require('./encoding')
+
+desired_high_water_mark = 4096
+calculate_high_water_mark = (input_length) ->
+  input_length*Math.floor(desired_high_water_mark/input_length)
```

Comment:
Just tested with defaults and it doesn't make an appreciable difference, removed in 766fe43f19493f589d9c0def12c5187062115c2c